module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -2071,13 +2075,15 @@ protected function performHoldRequest(array $requestBody): array
      * Get allowed service points for a request. Returns null if data cannot be obtained.
      *
      * @param string $instanceId  Instance UUID being requested
+     * @param string $itemId      Item UUID being requested
```

Comment:
This needs slight adjustment -- and then the lines above and below will need to be realigned to match.

```suggestion
     * @param ?string $itemId      Item UUID being requested (or null if unavailable/inapplicable)
```

Reply:
Good catch, I made that update.